src/Main.hs

```diff
@@ -13,19 +11,22 @@ main :: IO ()
 main = execParser opts >>= run
 
 run :: Config -> IO ()
-run (Config w p) = T.putStrLn . wrap w . parseFile =<< contentsOrSTDIN p
+run c = T.putStrLn . wrap c . parseFile =<< contentsOrSTDIN (path c)
 
 contentsOrSTDIN :: FilePath -> IO Text
 contentsOrSTDIN "-" = T.getContents
 contentsOrSTDIN p = T.readFile p
 
-wrap :: Int -> [Content] -> Text
-wrap w = T.stripEnd . T.unlines . concatMap (wrapContent w)
+wrap :: Config -> [Content] -> Text
+wrap c = T.stripEnd . T.unlines . concatMap (wrapContent c)
```

Comment:
I don't think so, those types don't line up. I'm passing `wrapContent c` to `concatMap` which is then taking `[Content]` as the second argument. I don't think there actually is a (reasonable) way to make this pointfree _or_ use `$` here.